components/pages/explorer/RepoTable/index.tsx

```diff
@@ -19,463 +19,277 @@
+
+	const theme = useTheme();
+	const { logEvent } = useTrackingContext();
+	const {
+		NEXT_PUBLIC_ARRANGER_API,
+		NEXT_PUBLIC_ARRANGER_MANIFEST_COLUMNS,
+		NEXT_PUBLIC_ENABLE_DOWNLOADS,
+		NEXT_PUBLIC_SINGULARITY_API_URL,
+	} = getConfig();
+
+	const [archive, setArchive] = useState<Archive | undefined>(undefined);
+	const [showDownloadInfoModal, setShowDownloadInfoModal] = useState(false);
+
+	const showModal = () => {
+		setArchive(undefined);
+		setShowDownloadInfoModal(true);
+	};
+	const closeModal = () => {
+		setShowDownloadInfoModal(false);
+		setArchive(undefined);
+	};
+
+	const saveSetThenBuildArchive = async (sqon: SQONType): Promise<Archive> => {
+		const setId = await saveSet(sqon);
+		logEvent({
+			category: 'Downloads',
+			action: 'Archive Build',
+		});
+		return await startArchiveBuildBySetId(setId);
+	};
+
+	const updateArchiveState = async () => {
+		if (archive?.status === 'BUILDING') {
+			await sleep(5000);
+			await findArchiveById(archive.id).then(setArchive);
+		} else if (archive?.status === 'COMPLETE' && showDownloadInfoModal) {
+			logEvent({
+				category: 'Downloads',
+				action: 'Archive Download',
+			});
+
+			window.location.assign(
+				urlJoin(NEXT_PUBLIC_SINGULARITY_API_URL, '/download/archive/', archive.id),
+			);
+		}
+	};
+
+	useEffect(() => {
+		updateArchiveState();
+	}, [archive]);
+
+	const handleBundleDownload: ExporterFunction = ({ selectedRows }) => {
+		showModal();
+
+		const sqonToUse = buildSqonWithObjectIds(sqon, selectedRows);
+
+		const archivePromise = isEmpty(sqonToUse)
+			? fetchLatestArchiveAllInfo()
+			: saveSetThenBuildArchive(sqonToUse);
+
+		archivePromise.then(setArchive);
+	};
+
+	const today = new Date().toISOString();
+	const tsvExportColumns = NEXT_PUBLIC_ARRANGER_MANIFEST_COLUMNS.split(',')
+		.filter((field) => field.trim()) // break it into arrays, and ensure there's no empty field names
+		.map((fieldName) => fieldName.replace(/['"]+/g, '').trim())
+		.map((fieldName): Partial<CustomColumnMappingInterface> => {
+			return {
+				displayFormat: ({ displayFormat = '', displayType = '', type = '' } = {}) => {
+					return displayFormat || ([displayType, type].includes('date') && 'yyyy-MM-dd') || null;
+				},
+				displayName: ({ displayName = '', Header = '' } = {}) => {
+					switch (fieldName) {
+						case 'study_id':
+							return fieldName;
+
+						default:
+							return (displayName || Header)
+								.toLowerCase()
+								.replace(/(\s+)ct(\s+)/g, '$1Ct$2')
+								.replace(/(\s+)id/g, '$1ID')
+								.replace(/(\s*)gisaid(\s*)/g, '$1GISAID$2');
+					}
+				},
+				fieldName,
+			};
+		});
+
+	const customExporters: CustomExporterInput = NEXT_PUBLIC_ENABLE_DOWNLOADS
+		? [
+				{
+					columns: tsvExportColumns,
+					fileName: `virusseq-metadata-export-${today}.tsv`,
+					function: 'saveTSV',
+					label: 'Metadata only',
+					valueWhenEmpty: '',
+				},
+				{ function: handleBundleDownload, label: 'Metadata & Fasta files' },
```

Comment:
```suggestion
				// { function: handleBundleDownload, label: 'Metadata & Fasta files' },
```

commenting this line out will remove the Singularity downloads temporarily